<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="1" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="23" started-at="2018-11-16T19:17:44Z" finished-at="2018-11-16T19:17:44Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="23" started-at="2018-11-16T19:17:44Z" finished-at="2018-11-16T19:17:44Z">
      <class name="com.julyanajs.testing.SkipTest">
        <test-method status="PASS" signature="firstTest()[pri:0, instance:com.julyanajs.testing.SkipTest@51081592]" name="firstTest" duration-ms="6" started-at="2018-11-16T19:17:44Z" finished-at="2018-11-16T19:17:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- firstTest -->
        <test-method status="SKIP" signature="secondtTest()[pri:0, instance:com.julyanajs.testing.SkipTest@51081592]" name="secondtTest" duration-ms="1" started-at="2018-11-16T19:17:44Z" finished-at="2018-11-16T19:17:44Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Login failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Login failed
at com.julyanajs.testing.SkipTest.secondtTest(SkipTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- secondtTest -->
      </class> <!-- com.julyanajs.testing.SkipTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
